### PR TITLE
#161457844 Write Tests for the endpoints 

### PR DESCRIPTION
**Pull request by Georgeygigz**
**What does this PR do?**
Writes tests for the endpoints

**Description completed tasks**
1. Writing failing tests for:
              _- create a new account and login_
              _-Invalid email, password_
              _-add  update and view products_
              _-add and view sales records_
              _-malformed urls_

**How to manually test this**
 1. Make a new directory on your computer
 2. Open the terminal and navigate to the folder
 3. `git clone` this  <code>[repo](https://github.com/Georgeygigz/store-manager-api/)</code>
 4.  run `pip install -r requirements.txt` to install the dependencies
 5.  Then on your terminal write `nosetests --with-coverage --cover-package=app && coverage report` to run the tests

**PT Stories**
#161457844 

**Screen shot**
image

### CODE REVIEW

File: app/tests/v2/test_store_views.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+    '''Test invalid post url'''
+
+    def test_invalid_post_product_url(self):
+        response=self.invalid_post_product_url()
+        self.assertEqual(response.status, '404 NOT FOUND')
+
+    def test_invalid_get_product_url(self):
+        response = self.invalid_get_product_url()
+        self.assertEqual(response.status, '404 NOT FOUND')
```
Comment (kwanj-k) [inline app/tests/v2/test_store_views.py]: I was adviced to live new lines
Add one here

Comment (dotNesh) [inline app/tests/v2/test_store_views.py]: Leave a blank line at the end of each file.

**Review by dotNesh — APPROVED** [review]
LGTM. Ensure to fix the one issue.

File: app/tests/v2/test_store_views.py (hunk 1 above)
Comment (Georgeygigz) [reply app/tests/v2/test_store_views.py]: Okay, I will ensure that

**Pull request merged**